.github/workflows/shellcheck-ci.yml

```diff
@@ -0,0 +1,30 @@
+name: ShellCheck CI
+run-name: ShellCheck CI triggered from @${{ github.actor }} of ${{ github.head_ref }}
+
+on:
+  workflow_dispatch:
+  merge_group:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  arbitrator:
```

Comment:
Maybe rename the job nome to shellcheck?

Reply:
Yep :), done